### DISCUSSION TITLE
Consideration for Combobox keepMounted default value

### DISCUSSION THREAD

**Discussion by jacobwilson41** [mantinedev/mantine#6227] [Feedback]
I would like to propose that the `keepMounted` prop's default value be `false` instead of `true`.

In the case where many combobox's are rendered and each one has many options, this will result in slow rendering performance.  

I just got done spending many hours debugging this situation and tracked the issue to the fact that each of my 17 combobox's each had to render their 192 options because of `keepMounted` default value of `true`.

Unfortunately I am building an application in an environment that does not have access to the React devtools and so it was difficult to find the cause of the issue.

I think its worth considering that the `keepMounted` option be opt-in instead of opt-out.

I can't think of any downsides to this proposal so maybe this community will show me the potential tradeoffs that I'm missing.

**Comment by rtivital** [comment]
I'll check whether is it possible to create an accessible solution for that.
